### PR TITLE
GH-1473 Update GitHub Pages with new Swagger access

### PR DESCRIPTION
**Pull request by okotsopoulos**
### Purpose

- https://broadinstitute.atlassian.net/browse/GH-1473

More context [here down in Slack](https://broadinstitute.slack.com/archives/CFA6Q2QBU/p1633100269359500?thread_ts=1633094768.352500&cid=CFA6Q2QBU).

### Changes

- Updated Developer Process docs to reflect new means of Swagger access -- simpler, now that there is no more UI!

### Review Instructions

I rendered a local docsite per the `readme` and verified that the instructions worked for local WFL off of latest `develop`.
```
make docs
. derived/.venv/docs/bin/activate && (  cd derived/docs; python3 -m mkdocs serve  )
```

**Note that the URLs given for WFL dev and prod Swaggers don't work!**  That's because we haven't released the UI removal change to either environment.

--

~Can anyone confirm the behavior around deploying changes to the docs -- will these get deployed immediately on merge to dev and prod docsites, or only on the releases to those environments?  That may impact my merge strategy.~

ETA: I observed after merging an unrelated doc change that when a doc change is merged, it is automatically deployed to dev and prod environments.  This was a change made when we were in the following situation:

- Release staged workload support for COVID sequencing in June '21, but no corresponding documentation - v0.7.0
- Soon after, complete the documentation
- Many months pass without a release and without public documentation to share, so Rex makes a change to automatically deploy doc changes on merge

We knowingly traded one problem -- documentation out of date -- for another -- documentation "ahead" of date.  Would be worth an engineering discussion for how our proposed release automation changes should also impact our release of the docsite.

In the meantime, I will try to hold off on merging this until immediately after the next release: https://broadinstitute.atlassian.net/browse/GH-1472

### CODE REVIEW

**Review by rfricke-asymmetrik — APPROVED** [review]

**Review by rhiananthony — APPROVED** [review]
👍🏼

File: docs/md/dev-process.md [hunk 1]
```diff
@@ -6,16 +6,12 @@ encourage ourselves to follow in most cases.
...
 
 WFL ships with a Swagger UI that documents all available endpoints. It's
-available at path `/swagger`.  At present, we cannot hit it directly
-without logging in first because it is bundled with the UI and not the API.
+available at path `<host>/swagger-ui/`.
 
-- Log into WFL UI, e.g. https://dev-wfl.gotc-dev.broadinstitute.org/login
-- Navigate to `/swagger` via Swagger API button in top right
+- [Dev WFL Swagger](https://dev-wfl.gotc-dev.broadinstitute.org/swagger-ui/)
+- [Prod WFL Swagger](https://prod-wfl.gotc-prod.broadinstitute.org/swagger-ui/)
```
Comment (tbl3rd) [inline docs/md/dev-process.md:12]: `/swagger`?

File: docs/md/dev-process.md [hunk 2]
```diff
@@ -240,28 +238,11 @@ open http://localhost:3000/swagger/swagger.json
...
-You need to start the UI server too
-if you want to see the swagger page rendered.
-With the local WFL server running on port 3000 as above,
-start the UI this way.
-
+To view the rendered Swagger page:
 ``` shell
-cd ./ui
-npm run serve
+open http://localhost:3000/swagger-ui/
```
Comment (tbl3rd) [inline docs/md/dev-process.md:243]: Same here.

File: docs/md/dev-process.md [hunk 3]
```diff
@@ -240,28 +238,11 @@ open http://localhost:3000/swagger/swagger.json
...
-With the UI running,
-open this URL in the browser.
-
-``` shell
-open http://localhost:8080/
-```
-
-Click the `LOGIN WITH GOOGLE` button if necessary,
-Then the `SWAGGER API` button at the upper right.
-You should see an interactive Swagger API page.
```
Comment (tbl3rd) [inline docs/md/dev-process.md:262]: Thanks.

File: docs/md/dev-process.md [hunk 4]
```diff
@@ -54,7 +50,7 @@ There is also a
 [Calva](https://marketplace.visualstudio.com/items?itemName=betterthantomorrow.calva)
 plugin for [Visual Studio Code](https://code.visualstudio.com/).
 
-I hack Clojure in Emacs using
+[Tom Lyons](mailto:tbl@broadinstitute.org) hacks Clojure in Emacs using
```
Comment (tbl3rd) [inline docs/md/dev-process.md:53]: Thanks.

File: docs/md/dev-process.md [hunk 5]
```diff
@@ -6,16 +6,12 @@ encourage ourselves to follow in most cases.
 ## The Swagger page
 
 WFL ships with a Swagger UI that documents all available endpoints. It's
-available at path `/swagger`.  At present, we cannot hit it directly
-without logging in first because it is bundled with the UI and not the API.
+available at path `<host>/swagger-ui/`.
```
Comment (tbl3rd) [inline docs/md/dev-process.md:9]: The URI is /swagger, isn't it?

**Review by tbl3rd — APPROVED** [review]

**Comment by okotsopoulos** [comment]
Now that WFL v0.9.24 has been released, updated to reflect final means of Swagger access and will merge.

**Pull request merged**